prioritization.md

```diff
@@ -0,0 +1,39 @@
+
+**Version:** 1.0
+
+**Last updated:** August 2020
+
+## Prioritization guidelines
+
+We prioritize the following principles when considering budget allocation:
+
+*   We consider community engagement to be very important. It is an indicator of long term sustainability.
+*   We like to help the enablers within communities. These people often take on the tedious but important jobs. Or they help tackle and resolve difficult problems. They may be both old timers or fresh and enthusiastic new contributors.
+*   We prioritize ideas which have wide support from within our community forums. Ideas need to be debated within our forums and have gathered support before they will be considered for sponsorship.
+*   We prioritize high value ideas, ones which lift the collective effectiveness of our community.
+*   We like unblocking obstacles preventing the community from moving forward. Maybe a technical challenge, maybe a bunch of boring grunt work or it could be business-as-usual overheads.
+*   We prioritize ideas which are not already being solved elsewhere.
+*   We are wary of introducing a maintenance liability. We want material which is going to be adopted and maintained by the greater community.
+*   We look favourably on matching contributions. If someone else is prepared to sponsor your idea, it is a good indicator of a worthiness.
+*   We like to help owners of useful proprietary material who wish to share it under an open license. This could be by covering the business costs of sharing, or by counting the material as an in-kind contribution.
+*   The results of any implementation, and its dependencies, must be available under the open licenses used by our project.
+
+## Sponsorship
+
+*   We welcome financial sponsorship of our project.
+*   We expect our sponsors to behave ethically and uphold business practices which align with our [Code of Conduct](https://github.com/thegooddocsproject/governance/blob/master/CodeOfConduct.md).
+*   Sponsors are encouraged to extend the value of their sponsorship by engaging with our community and providing co-mentors on initiatives.
+*   Sponsors can set guidelines on how their sponsorship is spent, such as targeting specific bounty projects. These guidelines will need to align with the goals of The Good Docs Project and the greater technical writing community.
+*   Note that sponsorship influence should be used responsibly. It should promote rather than hinder the needs and values of The Good Docs Project and the wider technical writing ecosystem.
+*   We appreciate the value provided by sponsors and will show sponsors on our website.
+
+## Thanks to our contributors
+
+We acknowledge the significant value created by contributions within our community. Many people have given as volunteers. Others are paid a fraction of their worth in the marketplace. For that we are privileged and thankful. 
```

Comment:
"Many people have given as volunteers"

Could this be better said as: "Many people have donated their time as volunteers" (or something similar).

Reply:
Agreed, I had to read it over a couple of times.

Reply:
Agree with Jared

Reply:
Agree with Jared here: Many people have offered/donated their time as volunteers